src/views/AccountSettings.js

```diff
@@ -52,40 +52,38 @@ function UserSettings({ history }) {
 
   useEffect(() => {
     const doFetch = async () => {
-      setIsError(false);
+      setErrorMessage(``);
       setIsLoading(true);
-      // checks if jwt is valid and hasn't expired
-      checkJwt(jwt);
+      // checks if auth is valid and hasn't expired
+      checkAuthExpiry(authExpiry);
 
       try {
         const result = await axiosWithCache.get(
-          `${API_ROOT}/profile?address=${userAddress}&jwt=${jwt}`
+          `${API_ROOT}/profile?address=${userAddress}`
         );
 
         profileDispatch({ type: "SET_PROFILE_DATA", payload: result.data });
       } catch (error) {
-        setIsError(true);
-        console.log(error);
+        setErrorMessage(``);
```

Comment:
`error.response.data` - add to the `setErrorMessage` hook in component state to be rendered